expertise/create_dataset.py

```diff
@@ -68,7 +68,9 @@ def get_paper_notes(self, author_id, dataset_params):
             return [n for n in self.openreview_client.get_notes_by_ids(ids=note_ids) if n.invitation == paper_invitation]
 
         notes_v1 = list(openreview.tools.iterget_notes(self.openreview_client, content={'authorids': author_id}))
+        notes_v1 = [note for note in notes_v1 if note.readers == ['everyone'] and  getattr(note, 'pdate', None)]
```

Comment:
do you all v1 notes have a pdate property @carlosmondra ? we might try removing them. 

Also if we are filtering out by pdate then the Expertise UI should do that same too. @enrubio can you make that change?

Reply:
> do you all v1 notes have a pdate property @carlosmondra ? we might try removing them.

No, not all notes in v1 have pdate even if they should. Notably imported v1 dblp notes may not have it. I wonder if we should migrate the v1 dblp notes to v2.

Reply:
> I wonder if we should migrate the v1 dblp notes to v2.
YEs!

Reply:
so we should remove `getattr(note, 'pdate', None)`?